crowdsec-docs/unversioned/console/cti/cve_explorer.md

```diff
@@ -0,0 +1,182 @@
+
+
+> **Try it now** → [https://app.crowdsec.net/cti/cve-explorer](https://app.crowdsec.net/cti/cve-explorer)
+
+  <div align="center">
+    <img src="/img/console/cti/cve-explorer/cve-home.png" alt="Screenshot — CVE Explorer home" width="70%" />
+  </div>
+
+---
+
+## CVEs Exploration
+
+### Search Bar
+  <div align="center">
+    <img src="/img/console/cti/cve-explorer/cve-search.png" alt="Screenshot — CVE Explorer search" width="50%" />
+  </div>
+
+
+At the top of the CVE Explorer sits a unified search field where you can instantly filter the entire CVE catalog by:
+
+* **CVE ID** (e.g. `CVE-2021-44228`)
+* **Product or Vendor** (e.g. Apache, Cisco)
+* **Attack Type** (e.g. RCE, XSS)
+
+Type your query, hit **Search**, and the list below updates in real-time.
+
+---
+
+### Quick-Filter Cards
+
+  <div align="center">
+    <img src="/img/console/cti/cve-explorer/cve-shortcuts.png" alt="Screenshot — CVE Explorer shortcuts" width="70%" />
+  </div>
+
+Directly under the search bar are three actionable buttons that sort all CVEs in the database:
+
+1. **Trending CVEs**
+
+   * Vulnerabilities with the fastest week-over-week growth in exploit detections.
+2. **Recently Tracked CVEs**
+
+   * Newly observed by CrowdSec for catching zero-day or freshly published ones.
+3. **Most Exploited CVEs**
+
+   * Ranked by total unique attacking IPs, showing the biggest exploits first.
+
+---
+
+### CVE Result List
+
+  <div align="center">
+    <img src="/img/console/cti/cve-explorer/cve-list-item.png" alt="Screenshot — CVE Explorer list" width="70%" />
+  </div>
+
+
+Beneath the filters, CVEs are presented as concise “cards” with these key data points:
+
+* **Title & ID** (click anywhere on the card to drill down)
+* **CrowdSec Score** (0–10 risk rating)
+* **Sparkline** of exploit volume over the last 7 days
+* **First Detection** date in CrowdSec’s network
+* **# of Exploiting IPs** badge on the right
+* **Product/Vendor** tags
+
+
+## CVE Details Page
+
+
+  <div align="center">
+    <img src="/img/console/cti/cve-explorer/cve-details-head.png" alt="Screenshot — CVE Details" width="70%" />
+  </div>
+
+The header zone tells you, at a glance, how dangerous the vulnerability is for your stack:
+
+
+* **CrowdSec Score** – CrowdSec proprietary risk rating (0-10)
+* **CVSS Base Score** – industry-standard severity
+* **Published / First Seen** – NVD publication date and the first time CrowdSec witnessed exploitation in the wild
+* **Exploiting IPs** – unique IPs detected attacking with this CVE
+* **Products / Vendors** – affected software stack
+
+> **Pro tip:** The *CrowdSec score* represents the calculation of how dangerous a given vulnerability is based on the attacks seen. \
+>  The computing of the value is based on:
+> - How recent the CVE is
+> - Solutions of mitigation 
+> - The trendiness of exploitation
+> - The severity 
```

Comment:
"The severity" is eyes of the beholder, is it based on CVE score, if so say that instead of "severity" as people class a 0.1 as severe